CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 ## Changed
 
+- [[#255](https://github.com/rust-vmm/kvm-ioctls/issues/255)]: Fixed a
+  soundness issue when accessing the `kvm_run` struct.
+
 # v0.16.0
```

Comment:
Could we list all the functions that had their signature changed here?

Reply:
Done